tests/test_alias.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2024, Basalte bv
```

Comment:
curious, why doesn't the files in `/tests` have a license tag :thinking:

Reply:
Added it to this file.